### PR TITLE
eos23041-Config phase validations

### PR DESCRIPTION
**Pull request by shalakadharap**
# Problem Statement
- Problem statement 

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/setup/openldap/openldap.py [hunk 1]
```diff
@@ -81,12 +81,13 @@ def validate(self, phase: str):
...
         return 0
 
-    def _key_value_verify(self, key: str):
+    def _key_value_verify(self, key: str, phase: str):
         """Verify if there exists a corresponding value for given key."""
 
         value = Conf.get(self.index, key)
         if not value:
-            raise OpenldapSetupError({"message":"Empty value for key"})
+            Log.debug("Validation failed\n")
```
Comment (bkirunge7) [inline py-utils/src/utils/setup/openldap/openldap.py:89]: please log key and self.index that will help to debug

File: py-utils/src/utils/setup/openldap/openldap.py [hunk 2]
```diff
@@ -129,16 +131,21 @@ def _expand_keys(self, key: str):
         if self.cluster_id is not None:
             storage_set_count_key = storage_set_count_key.\
                 replace("cluster-id", cluster_id_val)
-            storage_set_count_str = Conf.get(self.index, storage_set_count_key)
+            try:
+                storage_set_count_str = Conf.get(self.index, storage_set_count_key)
+            except:
+                Log.debug("Validation failed\n")
```
Comment (bkirunge7) [inline py-utils/src/utils/setup/openldap/openldap.py:137]: please log variables

File: py-utils/src/utils/setup/openldap/openldap.py (hunk 1 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/openldap.py:89]: Fixed

File: py-utils/src/utils/setup/openldap/openldap.py (hunk 2 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/openldap.py]: done

**Review by bkirunge7 — APPROVED** [review]

File: py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.1-node [hunk 3]
```diff
@@ -2,10 +2,6 @@ server_node:
   TMPL_MACHINE_ID:
     cluster_id: "TMPL_CLUSTER_ID"
     hostname: "TMPL_HOSTNAME"
-    network:
```
Comment (vimalraghwani20) [inline py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.1-node:5]: remove same from the init phase if not required

Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/templates/openldap.config.tmpl.1-node:5]: for init phase, let @shahidSTX  decide as he is working on it.
I am taking care of Test and COnfig

**Pull request merged**